libr/bin/pdb/pdb.c

```diff
@@ -802,7 +802,7 @@ void build_command_field(ELeafType lt, char **command_field) {
 		if (!(*command_field)) {
 			break;
 		}
-		strcpy (*command_field, "pf.");
+		strcpy (*command_field, "pf");
```

Comment:
Are you sure about that ? Can you provide a testcase?

Reply:
https://gist.github.com/anonymous/57e419726335f709d6d00cd733364408
Here is a scripted gdb session that shows a memory corruption if the patch will not be applied.

Reply:
No but the pf and pf. is supposed to have a purpose it's not about a memory corruption, if you break the tool for fixing memory corruption it's pointless :/

Reply:
Well then we should allocate the memory for pf. not pf. The problem is that memory doesn't much the amount of bytes copied. And alocate smaller amount than we use.)